administracion/views.py

```diff
@@ -1,40 +1,63 @@
 from django.contrib.auth.models import User
 from django.shortcuts import render, redirect
-from django.contrib.auth.decorators import user_passes_test, login_required
-from .forms import FormaCreacionUsuario
+from django.contrib.auth.decorators import user_passes_test
+from .forms import UserForm
 from perfiles_usuario.utils import is_administrador
 
 
-@login_required
```

Comment:
Login required decorator is a wrapper of user passes test in which the user is authenticated. https://docs.djangoproject.com/en/1.10/_modules/django/contrib/auth/decorators/#login_required
I thought this was enough that is why I removed it, but I will put it again since it is a different condition which is being tested.